src/dev-app/system-config-tmpl.js

```diff
@@ -8,68 +8,10 @@
 
 // Note that this file isn't being transpiled so we need to keep it in ES5.
 
-var CDK_PACKAGES = [
-  'a11y',
-  'accordion',
-  'bidi',
-  'clipboard',
-  'coercion',
-  'collections',
-  'drag-drop',
-  'keycodes',
-  'layout',
-  'observers',
-  'overlay',
-  'platform',
-  'portal',
-  'scrolling',
-  'stepper',
-  'table',
-  'text-field',
-  'tree',
-];
-
-var CDK_EXPERIMENTAL_PACKAGES = [
-  'dialog',
-  'popover-edit',
-  'scrolling',
-];
-
-var MATERIAL_PACKAGES = [
-  'autocomplete',  'badge',
-  'bottom-sheet',  'button',
-  'button-toggle', 'card',
-  'checkbox',      'chips',
-  'core',          'datepicker',
-  'dialog',        'divider',
-  'expansion',     'form-field',
-  'grid-list',     'icon',
-  'input',         'list',
-  'menu',          'paginator',
-  'progress-bar',  'progress-spinner',
-  'radio',         'select',
-  'sidenav',       'slide-toggle',
-  'slider',        'snack-bar',
-  'sort',          'stepper',
-  'table',         'tabs',
-  'toolbar',       'tooltip',
-  'tree',
-];
-
-var MATERIAL_EXPERIMENTAL_PACKAGES = [
-  'mdc-button',
-  'mdc-card',
-  'mdc-checkbox',
-  'mdc-chips',
-  'mdc-tabs',
-  'mdc-helpers',
-  'mdc-menu',
-  'mdc-radio',
-  'mdc-progress-bar',
-  'mdc-slide-toggle',
-  'mdc-slider',
-  'popover-edit',
-];
+var CDK_PACKAGES = CDK_ENTRYPOINTS_TMPL;
```

Comment:
Should we include some kind of syntax show that these template markers are going to get replaced?